### ISSUE TITLE
new tensorboard add_scalar not allowing merged scalars on same chart

### ISSUE THREAD

**Issue by gatapia** [pytorch/pytorch#20092]
## To Reproduce

1. call add_scalar('\<groupname\>/\<id\>', scalar_value, epoch)
2. open tensorboard 

## Expected behavior

When using '\<group\>/\<id\>' as the tag for a scalar we should see the scalars plotted on the same chart.  This allows for train and test loss to be plotted on same chart for instance.  Instead we get a new header \<group\> with two charts \<id\>.

Note: My assumption is that this is the way tensorflowX used to work.

## Environment
 - PyTorch Version (e.g., 1.0): 1.1
 - OS (e.g., Linux): Linux
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.6
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

## Additional context
I also tried the add_scalars using multiple variation of tag names and dict keys to try to get the grouping to work as expected.  None of these worked.

**Comment by ezyang** [comment]
cc @lanpa

**Comment by lanpa** [comment]
That is expected. Instead, you should use `add_scalars` to get the graph you want.
The doc for pytorch repo is under review, please check:
https://tensorboardx.readthedocs.io/en/latest/tensorboard.html#tensorboardX.SummaryWriter.add_scalars

**Comment by gatapia** [comment]
Thanks, works as described.
